bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -320,4 +335,15 @@ private void returnToPool(Channel channel) {
       ((PooledChannel) channel).returnToPool();
     }
   }
+
+  private void expandPoolIfNecessary(int channelCount) {
+    if (!asyncExpantionExceptionOccurred.get()) {
+      try {
+        this.channelPool.ensureCapacity(channelCount);
+      } catch (IOException e) {
+        LOG.info("Could not expand the channel pool.", e);
```

Comment:
what possible error we could encounter when we see IOException when expanding the channel pool?